shogun-lib/src/main/java/de/terrestris/shogun/lib/event/KeycloakEventType.java

```diff
@@ -24,5 +24,7 @@ public enum KeycloakEventType {
     USER_GROUP_MEMBERSHIP_CHANGED,
     GROUP_CREATED,
     GROUP_DELETED,
-    GROUP_ROLES_CHANGED
+    GROUP_ROLES_CHANGED,
+    REALM_ROLE_CREATED,
+    REALM_ROLE_DELETED
```

Comment:
A bit of topic, but do we have to handle something like "USER_REGISTERED"?

Reply:
Good point, this should be handled as well. Since it's not directly related to to the newly added models I would favour to handle it in a separate PR, ok?

Reply:
Sure. It just came to me mind so i wanted to leave a note on this :)